src/tilt-in.css

```diff
@@ -0,0 +1,12 @@
+
+/* tilt in animation */
+.tilt-in-enter,
+.tilt-in-leave-to{
+    transform: rotateY(20deg) rotateX(35deg) translate(300px, -300px) skew(-35deg, 10deg);
+    opacity: 0;
+}
+
+.tilt-in-enter-active,
+.tilt-in-leave-active{
+    transition:  transform 0.3s, opacity 0.5s ;
```

Comment:
The transform duration here is little as well, making the animation too fast, please.

also, try adding `transition-timing-function` to make the animation slow down a bit towards the end.